src/code/z_viscvg.c

```diff
@@ -1,70 +1,100 @@
-#include "global.h"
+/**
+ * @file z_viscvg.c
+ * Description: Visualise Coverage (how much of the neighbourhood of a pixel is covered by a primitive) in various ways.
+ *
+ * To understand this file, it is helpful to remember that A_MEM is essentially synonymous with coverage, and that
+ * `GBL_c1/2(p, a, m, b)` are usually `(p * a + m * b) / (a + b)`.
+ *
+ * Coverage is maximum when not on an edge, on an edge it is smaller, and since coverage is treated as an alpha edges
```

Comment:
It is correct to say coverage is full outside of edges, but it's not necessarily low at edges.
It often is, but imagine an edge drawn almost perfectly in between two columns of pixels, both sides will have full coverage

Maybe instead of going into details you could rephrase like "coverage is typically low at edges" (or "often" instead of "typically" idk)

(btw I think "full" is the proper term for refering to the coverage being at its maximum, but you may want to check the manual on that)